NDcPP_v2_2e.adoc

```diff
@@ -3603,24 +3658,42 @@ FIA_X509_EXT.1 X.509 Certificate Validation
 
-*FCS_DTLSS_EXT.1.2* The TSF shall deny connections from clients requesting _[assignment: list of protocol versions]_.
++
+] and no other ciphersuites.
+
+*FCS_DTLSS_EXT.1.2* The TSF shall not proceed with a connection handshake attempt if the DTLS server cannot successfully validate the cookie returned by the DTLS Client.
+
+*FCS_DTLSS_EXT.1.3* The TSF shall authenticate itself using X.509 certificate(s) using [selection: _RSA with key size [selection: 2048 bits, 3072 bits, 4096 bits]; ECDSA over NIST curves [selection: secp256r1, secp384r1, secp521r1] and no other curves_].
 
-*FCS_DTLSS_EXT.1.3* The TSF shall not proceed with a connection handshake attempt if the DTLS Client fails validation.
+*FCS_DTLSS_EXT.1.4* The TSF shall perform key exchange using: [selection:
+
+* _RSA key establishment with key size [selection: 2048 bits, 3072 bits, 4096 bits];_
+* _EC Diffie-Hellman key agreement over NIST curves [selection: secp256r1, secp384r1, secp521r1] and no other curves;_
+* _Diffie-Hellman parameters [selection: of size 2048 bits, of size 3072 bits, of size 4096 bits, of size 6144 bits, of size 8192 bits, ffdhe2048, ffdhe3072, ffdhe4096, ffdhe6144, ffdhe8192]_
 
-*FCS_DTLSS_EXT.1.4* The TSF shall perform key establishment for TLS using [selection: _RSA with key size [selection: 2048 bits, 3072 bits, 4096 bits], Diffie-Hellman parameters with size_ [selection__: 2048 bits, 3072 bits, 4096 bits, 6144 bits, 8192 bits], Diffie-Hellman groups__ [selection__: ffdhe2048, ffdhe3072, ffdhe4096, ffdhe6144, ffdhe8192, no other groups], ECDHE curves [selection: secp256r1, secp384r1, secp521r1] and no other curves__].
+].
 
 *FCS_DTLSS_EXT.1.5* The TSF shall [selection: _terminate the DTLS session, silently discard the record_] if a message received contains an invalid MAC.
 
-*FCS_DTLSS_EXT.1.6* The TSF shall detect and silently discard replayed messages for:
+*FCS_DTLSS_EXT.1.6* The TSF shall detect and silently discard replayed messages for: 
 
 * DTLS records previously received.
 * DTLS Records too old to fit in the sliding window.
 
-*FCS_DTLSS_EXT.1.7* The TSF shall support [selection: _no session resumption or session tickets, session resumption based on session IDs according to RFC 4346 (TLS1.1) or RFC 5246 (TLS1.2), session resumption based on session tickets according to RFC 5077_].
+
+*FCS_DTLSS_EXT.1.7* The TSF shall support [selection: _no session resumption or session tickets, session resumption based on session IDs according to RFC4346 (TLS1.1) or RFC5246 (TLS1.2), session resumption based on session tickets according to RFC5077 (TLS1.2), session resumption according to RFC8446 (TLS1.3)_].
+
+*FCS_DTLSS_EXT.1.8* The TSF [selection: _provides, does not provide_] the ability to configure the list of supported ciphersuites as defined in FCS_DTLSS_EXT.1.1.
+
+*FCS_DTLSS_EXT.1.9* The TSF shall prohibit the use of the following extensions: [assignment: [.underline]#list of prohibited DTLS extensions#].
```

Comment:
@mclearn This SFR is missing the bullet item, and same comment as before... 'brackets and underlined text are not included in the MS Word doc'.

Reply:
Reviewed formatting options and altered to be a simple assignment without underlining.

Reply:
I think this can be resolved based on the associated comment earlier on agreeing to italics.

Reply:
Agreed